modules/firebase/go.mod

```diff
@@ -0,0 +1,83 @@
+module github.com/testcontainers/testcontainers-go/modules/firebase
+
+go 1.23.0
+
+require (
```

Comment:
suggestion: just before submitting the changes, run `go mod tidy` from the module dir to make sure all dependencies are in sync with the core. Else we'll receive an extra PR from dependabot 😅

Reply:
`go mod tidy` returned no changes. Was this expected?